search-app/src/cmr/search/services/query_execution/granule_counts_results_feature.clj

```diff
@@ -83,16 +83,18 @@
 
 (defn- extract-spatial-operator
   "Determine whether spatial conditions should be ORed or ANDed together"
-  [query]
-  (let [conditions (get-in query [:condition :conditions])
-        condition-is-shape? (->> conditions
-                                 (map keys)
-                                 flatten
-                                 (filter #(= % :shape))
-                                 seq)
-        parent-group-operator (get-in query [:condition :operation])]
-    (when condition-is-shape?
-      parent-group-operator)))
+  [context]
+  (if (empty? (:query-string context))
```

Comment:
I was going to ask if `(not (seq ..)` would be better here, but it turns out that's exactly how `empty?` is defined.